src/indexer/index.ts

```diff
@@ -39,6 +40,12 @@ init()
 async function init(): Promise<Array<DomainIndexer>> {
   const sharedConfig = await getSharedConfig(process.env.SHARED_CONFIG_URL!)
 
+  const coinMarketCapServiceInstance = new CoinMarketCapService(
+    process.env.COINMARKETCAP_API_KEY as string,
+    process.env.COINMARKETCAP_API_URL as string,
+    JSON.parse(process.env.TOKEN_SYMBOLS!) as Array<{ id: number; symbol: string }>,
```

Comment:
Dont we have symbols in shared config?

Reply:
yeah, I thought could be easier to have a map instead of iteraring over shared config to get the symbols. I will changed it to use shared config instead